toolkit/tools/imagecustomizer/container/test-mic-container.sh

```diff
@@ -0,0 +1,36 @@
+# Test container by running run.sh script inside it.
+set -eux
+
+containerTag=$1
+
+outputImage=$(readlink -f "../../out/containertestoutput.vhdx")
+outputImageDir=$(dirname $outputImage)
```

Comment:
nit: It is not a bad idea to add quotation marks around variables and commands to avoid whitespace being handled incorrectly.

e.g. `outputImageDir="$(dirname "$outputImage")"`

Similar comment elsewhere.